### PR TITLE
mp-units 2.4.0 support added

### PR DESCRIPTION
**Pull request by mpusz**
### Summary
Changes to recipe:  **mp-units/2.4.0**

#### Motivation
New release

#### Details
Just a new release.

---
- [x] Read the [contributing guidelines](https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md)
- [x] Checked that this PR is not a duplicate: [list of PRs by recipe](https://github.com/conan-io/conan-center-index/discussions/24240)
- [x] Tested locally with at least one configuration using a recent version of Conan

### CODE REVIEW

File: recipes/mp-units/config.yml [hunk 1]
```diff
@@ -1,4 +1,6 @@
 versions:
+  "2.4.0":
+    folder: 2.4.0
```
Comment (jcar87) [inline recipes/mp-units/config.yml:3]: Thanks @mpusz !
looks like this folder doesn't exist, is this mean to be a new folder or to reuse 2.3.0?

Comment (mpusz) [reply recipes/mp-units/config.yml:3]: Sorry, my fault :-(

Comment (mpusz) [reply recipes/mp-units/config.yml:3]: Fixed!

**Review by jcar87 — APPROVED** [review]

**Review by franramirez688 — APPROVED** [review]

**Review by uilianries — APPROVED** [review]
LGTM. No breaking changes: https://github.com/mpusz/mp-units/compare/v2.3.0...v2.4.0#diff-148715d6ea0c0ea0a346af3f6bd610d010d490eca35ac6a9b408748f7ca9e3f4

**Pull request merged**